### ISSUE TITLE
Joining the default group chat of a circle with existing chat history shows the "cant decrypt " omemo message

### ISSUE THREAD

**Issue by decentral1se** [snikket-im/snikket-server#117]
Unsure if this is a limitation of omemo or clients or server or whatever (excuse my ignorance) but this impacts UX greatly when joining and seeing all these warnings. Is there anything to be done here to improve the onboarding experience?

**Comment by mwild1** [comment]
Yes, I've been thinking about this recently. I think the cleanest thing we can do right now is not show the history at all (because it is not decryptable by newcomers). Whether that's best solved client or server side I haven't decided yet. But it's definitely a UX issue I would like to fix.

**Comment by decentral1se** [comment]
> I think the cleanest thing we can do right now is not show the history at all

This would be fantastic.

**Comment by decentral1se** [comment]
Precedent in synapse, for the room admin settings:

![image](https://user-images.githubusercontent.com/1991377/152863607-20c73063-afeb-4198-8c0d-f2f61c7957c8.png)
